src/Generator/GeneratorCommand.php

```diff
@@ -84,6 +87,18 @@ public function getDefaultFileName(): string
         return 'Default' . $this->getFileTypeCapitalized();
     }
 
+    protected function setOptions(): void
+    {
+        $optionsFromConfig = Config::get('apiato.generator-commands.options', []);
+
+        foreach ($optionsFromConfig as $key => $value) {
```

Comment:
only specific options should be loaded from config, we discovered the wether to generate tests or not should be the first one.